sechub-doc/src/docs/asciidoc/documents/code2doc/client/02_sechub_client.adoc

```diff
@@ -96,10 +96,11 @@ SECHUB_APITOKEN=${apitoken}
 sechub -jobUUID ${jobUUID} getReport
 ----
 
+
 [[section-client-false-positives-mark]]
 ===== markFalsePositives
-Read your report carefully and check which of the findings are really so called "false positives".
-After this create an json file containing jobUUID and ID of finding you decided to be an FP. If 
+Read your report carefully and check which of the findings are really so called "false positives" (FP).
```

Comment:
"really so called" is informal English. You could simply reduce it to `the findings are really "false positives" (FP)`.

Reply:
Fixed